src/_P016_IR.ino

```diff
@@ -417,12 +417,12 @@ boolean Plugin_016(byte function, struct EventStruct *event, String &string)
 
         if (nullptr != P016_data) {
           // convert result to uint32_t
-          uint32_t iCode = ((uint32_t) results.decode_type) * 0x1000000;  // Bits 31-24 (upper byte) for decode_type
+          uint32_t iCode = (static_cast<uint32_t> (results.decode_type)) * 0x1000000;  // Bits 31-24 (upper byte) for decode_type
```

Comment:
Isn't it more readible to perform a bitshift here?
Multiplying by a hex number is already unsigned, so it won't be more safe or guaranteed to be correct, just better readible.

Reply:
It looked a bit wonky from the first time I looked at it, I'll fix that